src/LatteContext/CollectedData/CollectedMethodCall.php

```diff
@@ -15,11 +15,14 @@ final class CollectedMethodCall extends CollectedLatteContextObject
     public const TERMINATING_CALL = 'terminating';
     public const OUTPUT_CALL = 'output';
 
-    private string $callerClassName;
+    /**
+     * @var ?class-string
+     */
+    private ?string $callerClassName;
```

Comment:
why nullable?

Reply:
Theoretically it can be called outside class scope. We used '' for it originally but nullable is imho better.

Reply:
I see it now, ok null it is